reactor-netty-http/src/main/java/reactor/netty/http/client/HttpClientOperations.java

```diff
@@ -534,7 +535,7 @@ protected void onOutboundComplete() {
 		}
 		else if (markSentBody()) {
 			//"FutureReturnValueIgnored" this is deliberate
-			channel().writeAndFlush(LastHttpContent.EMPTY_LAST_CONTENT);
+			channel().writeAndFlush(new EmptyLastHttpContent(channel().bufferAllocator()));
```

Comment:
same question here, can we cache the EmptyLastHttpContent in a static variable ?